### PR TITLE
Sync public folder

### PR DESCRIPTION
**Pull request by andrewxhill**
- [x] Will need this out first https://github.com/textileio/go-textile/pull/695
- [x] Then the removal of the local mod replacement

Work to make it better, jotting these down while they are on my mind:

- [ ] Dir based Thread schema. Easy to swap that in once we have it.
- [ ] Lots of little logic additions to improve when/how it syncs
- [ ] Should we blocks.remove() the previous addition to clean up storage?
- [x] Drop IPNS and possibly move to a gateway service for resolution of a Contact's www. 
- [x] Swap in the instructions + nice interface for the initial index.html template

### CODE REVIEW

**Comment by andrewxhill** [comment]
Okay, dropped IPNS and just uses a `post-update` script to do whatever the user wants. The default script updates a cloudflare record to the latest hash. 

Instructions to set that are: 

Create env variables for your cloudflare domain. So for example, mine are,

```bash
export PAGES_CLOUDFLARE_API_KEY=<key>
export PAGES_CLOUDFLARE_EMAIL=account@email.com
export PAGES_DOMAIN_NAME=textile.xyz
export PAGES_IPFS_RECORD=demo.textile.xyz
```

As long as those are set, if you modify your `index.html` file, found in `~/Library/Application\ Support/Textile/<acct>/public`, your cloudflare dns record will update. 

You can change the script to do whatever you want with a new index.html has, the editable script is just found in, `~/Library/Application\ Support/Textile/<acct>/scripts`

**Comment by andrewxhill** [comment]
feels good, happy to merge this one and move forward with next PRs

**Review by carsonfarmer — APPROVED** [review]
This is dang cool @andrewxhill! Happy to merge this and start using 💪

**Comment by andrewxhill** [comment]
cool, just need a new go-textile rc and i'll merge!

File: src/Stores/index.ts [hunk 1]
```diff
@@ -207,18 +207,21 @@ export class AppStore implements Store {
...
-    const { emitter } = textile.account.sync(true)
-    emitter.on('textile.snapshots.found', (item: any) => {
-      const opts: NotificationOptions = {
-        body: item.id,
-        timestamp: moment().unix()
-      }
-      const note = new Notification('Found and applying snapshot', opts)
-    })
+    textile.account.sync(true)
+      .then((stream: ReadableStream) => {
```
Comment (andrewxhill) [inline src/Stores/index.ts:211]: @carsonfarmer going to move forward with this PR but just FYI this is an update I made to accommodate the new streams. it's worth a new review of these 10 lines

**Pull request merged**